jsconfig.json

```diff
@@ -4,6 +4,7 @@
 		"module": "ES6",
 		"jsx": "preserve",
 		"checkJs": true,
+		"allowUnreachableCode": true
```

Comment:
Added this because VSCode kept removing code during testing, like if I used an early `return` or commented something out. This doesn't stop ESLint from recognizing the issue, it just stops VSCode from silently obliterating code during a save.